client/src/components/SliderContainer/SliderContainer.css

```diff
@@ -0,0 +1,77 @@
+.SliderContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  font-family: var(--default-font);
+  width: 90%;
```

Comment:
There's something going on with the width styling here - it looks responsive, but I think the `width` and sizing just needs some polishing:

<img width="1552" alt="Screen Shot 2019-12-11 at 10 10 05 AM" src="https://user-images.githubusercontent.com/4991553/70633436-90034d00-1bfe-11ea-8899-23ebd0aae2da.png">

<img width="1552" alt="Screen Shot 2019-12-11 at 10 11 26 AM" src="https://user-images.githubusercontent.com/4991553/70633484-a3aeb380-1bfe-11ea-9ad8-cf6a33165c65.png">
<img width="1552" alt="Screen Shot 2019-12-11 at 10 11 31 AM" src="https://user-images.githubusercontent.com/4991553/70633485-a3aeb380-1bfe-11ea-84fe-f8d510d74f90.png">